packages/react/src/__tests__/codemod.js

```diff
@@ -0,0 +1,201 @@
+// jscodeshift can take a parser, like "babel", "babylon", "flow", "ts", or "tsx"
```

Comment:
remove?

Reply:
test failures seem to be just from trying to run this as a test

Reply:
Whoops 😄